schema/one-time-polled-screening.json

```diff
@@ -36,7 +36,7 @@
                         "signatureAuth": []
                     }
                 ],
-                "summary": "Configure how PassFort will interact with your integration",
+                "summary": "Configure how Passfort will interact with your integration",
```

Comment:
Change to present tense: "Configure out Passfort interacts with your integration"